util/src/main/java/com/farao_community/farao/util/NativeLibraryLoader.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ *  License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.farao_community.farao.util;
+
+import com.farao_community.farao.commons.FaraoException;
+
+/**
+ * @author Joris Mancini {@literal <joris.mancini at rte-france.com>}
+ */
+public final class NativeLibraryLoader {
```

Comment:
is it really useful?

Reply:
Not much but maybe we will keep to avoid struggling too much next time we face the problem !